odoo/openupgrade/doc/source/modules100-110.rst

```diff
@@ -469,7 +469,7 @@ missing in the new release are marked with |del|.
 +-----------------------------------+-----------------------------------+
 |sale_timesheet                     |                                   |
 +-----------------------------------+-----------------------------------+
-|sales_team                         |                                   |
+|sales_team                         | done                              |
```

Comment:
That needs to be a Capital D in (D)one, otherwise our CI does not know that this module is migrated.